doc/releases/release-notes-4.2.rst

```diff
@@ -246,6 +246,10 @@ New Boards
 
    * :zephyr:board:`cygnet` (``cygnet``)
 
+* Elements
```

Comment:
If the vendor of the board is actually "elements" then please put the board in an "elements" folder and not "others". However I couldn't find any reference online to such a vendor?

Reply:
`elements` is not a vendor, it's a project with open source IPs and SoC builder similar to `litex`. `aesc` would be the actual vendor.

Reply:
The board should reflect the board vendor. I take it this is the vendors repo? https://github.com/aesc-silicon/ElemRV

Reply:
@henrikbrixandersen yes it is. Updated to `aesc`.